patches/ui-webui-resources-cr_elements-cr_button-cr_button.css.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/ui/webui/resources/cr_elements/cr_button/cr_button.css b/ui/webui/resources/cr_elements/cr_button/cr_button.css
+index 78d85bb2a0eb81c56f1b2675618402b187715cec..933533071e100aecdd9898cf8df9fcd6bd47320a 100644
+--- a/ui/webui/resources/cr_elements/cr_button/cr_button.css
++++ b/ui/webui/resources/cr_elements/cr_button/cr_button.css
+@@ -360,3 +360,48 @@ paper-ripple {
+ :host-context([chrome-refresh-2023]) paper-ripple {
+   z-index: 1;
+ }
++
++:host(:not([style])) {
```

Comment:
Do we need the `:not([style])`? what if it's used with a `style="margin-top: 10px"`, do we want to not apply our overrides?

Reply:
We had a chat in Slack - this is the existing behavior, taken from the existing overrides.

Not sure why it's doing this, but let's not change things in release.